buildbot_gitea/reporter.py

```diff
@@ -25,7 +25,7 @@
 class GiteaStatusPush(http.HttpStatusPushBase):
     name = "GiteaStatusPush"
     neededDetails = dict(wantProperties=True)
-    ssh_url_match = re.compile(r"(ssh://)?[\w+\-\_]+@[\w+\.\-\_]:?\d*/(?P<owner>[\w_\-\.]+)/(?P<repo_name>[\w_\-\.]+)\.git")
+    ssh_url_match = re.compile(r"(ssh://)?[\w+\-\_]+@[\w\.\-\_]+:?(\d*/)?(?P<owner>[\w_\-\.]+)/(?P<repo_name>[\w_\-\.]+)(\.git)?")
```

Comment:
The first part of the changes makes sense, but i'm not sure why the .git suffix should be optional, does gitea not always include them ?